packages/fxa-auth-server/lib/server.js

```diff
@@ -40,6 +43,15 @@ function trimLocale(header) {
   return str.trim();
 }
 
+function logValidationError(response, log) {
+  if (response?.__proto__.name !== 'ValidationError') {
+    return;
+  }
+
+  log.error('server.ValidationError', response);
+  reportValidationError(response.stack, response);
```

Comment:
Looking at the implementation in the reportValidationError function, I am noticing that the validation error details should be set in the [scope context](https://github.com/mozilla/fxa/blob/d1fffc03697aca9e683ba03826d56d6526e62cd7/packages/fxa-shared/sentry/report-validation-error.ts#L53), however, in Sentry I don't seem to see these error details. 

I wonder if they are coming through? The error details should be an array of issues with validation, is why I am bringing this up.

Reply:
Maybe we can expand them a bit to ensure we get them all? Or maybe it doesn't like arrays.